test/parallel/test-assert.js

```diff
@@ -406,7 +406,7 @@ assert.doesNotThrow(function() { assert.ifError(); });
 
 assert.throws(() => {
   assert.doesNotThrow(makeBlock(thrower, Error), 'user message');
-}, /Got unwanted exception: user message/,
+}, /Got unwanted exception:\ntest: user message/,
               'a.doesNotThrow ignores user message');
```

Comment:
@jasnell `a.doesNotThrow ignores...` should be aligned with `/Got unwanted exception...`. Do I understand it correctly?